app/views/play/level/ControlBarView.coffee

```diff
@@ -159,7 +159,7 @@ module.exports = class ControlBarView extends CocoView
     @openModalView gameMenuModal
     @listenToOnce gameMenuModal, 'change-hero', ->
       @setupManager?.destroy()
-      @setupManager = new LevelSetupManager({@supermodel, @level, @levelID, parent: @, @session, @courseID, @courseInstanceID})
+      @setupManager = new LevelSetupManager({@supermodel, @level, @levelID, parent: @, @session, @courseID, @courseInstanceID, classroomItems: @classroom.get('classroomItems', true)})
```

Comment:
what if classroom is undefined/not present in that view like teacher?

Reply:
it actually failed here during my testing for home user: classroomItems: this.classroom.get('classroomItems', true)
Cannot read properties of undefined (reading 'get')
    at ControlBarView.eval (ControlBarView.coffee:262:40)

I made this comment based on just code 😅